src/js/containers/award/AwardInfoContainer.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+  * AwardInfoContainer.jsx
+  * Created by Emily Gullo 01/20/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import moment from 'moment';
+
+import SummaryBar from 'components/award/SummaryBar';
+import AwardInfoBar from 'components/award/AwardInfoBar';
+
+import * as awardActions from 'redux/actions/award/awardActions';
+
+const propTypes = {
+    getSelectedAward: React.PropTypes.func
+};
+
+class AwardInfoContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
As discussed in person, this component's state can be simplified. Most of the state appears to be simply string display of data. Additionally, tracking the award/funding agency toggle can be done its own component (`AgencyInfo`).